app/src/modules/settings/routes/flows/components/logs-drawer.vue

```diff
@@ -4,14 +4,15 @@ import { computed, toRefs, unref } from 'vue';
 import { useI18n } from 'vue-i18n';
 import { getTriggers } from '../triggers';
 import { FlowRaw } from '@directus/types';
-import { useRevision } from '@/composables/use-revision';
+import VDetail from '@/components/v-detail.vue';
 
 const props = defineProps<{
-	revisionId?: number;
 	flow: FlowRaw;
+	revision: any;
```

Comment:
Use the existing `Revision` type from app/src/types/revisions.ts

```suggestion
	revision: Revision;
```

Reply:
done